### PR TITLE
created finite-fault module skeleton and implemented sharedProductCon…

### LINKED ISSUE #1039: Finite Fault Module
Skeleton including Images in product with non event specific text.

### PR DESCRIPTION
**Pull request by cswilson-usgs**
Fixes #1039

### CODE REVIEW

File: src/app/finite-fault/finite-fault/finite-fault.component.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <h2>
+      <abbr title="Finite Fault">
+          Finite Fault
+      </abbr>
+    </h2>
+  </ng-container>
+
+  <div *ngIf="eventService.product$ | async as product">
+
+    <h2>Result</h2>
```
Comment (ehunter-usgs) [inline src/app/finite-fault/finite-fault/finite-fault.component.html]: We have the `shared-text-product` that allows you to read in text-like products. You can use this component to display the finite-fault html. The html is referenced using the `product.properties.eventsourcecode + '.html'` and is found in `product.contents` (you don't need to know that last part, but if you want to see the path to the html file that's where you find it).

finite-fault.component.html should look something like this...
```
  <div *ngIf="eventService.product$ | async as product">
    <shared-text-product
        [contentPath]="product?.properties?.eventsourcecode + '.html'"
        [product]='product'>
    </shared-text-product>
  </div>
```

Comment (jmfee-usgs) [reply src/app/finite-fault/finite-fault/finite-fault.component.html]: @ehunter-usgs : we're moving away from the free-form product, into one more structured for consistency...  The shared-text-product will still be used, but for the empty-path content in a future product.

**Comment by ghayes-usgs** [comment]
Yes, information in results section differs for each event, as you've indicated. The 4 parameters listed are sufficient.
I am happy for static waveform images to not be shown, as long as they are included in the Downloads section.

**Comment by emartinez-usgs** [comment]
- Use dot-notation when accessing object properties whenever possible.
- Use Elvis-operator for safer object property chain access.
- Were surface projection and cross section slip sections swapped? Why?
- Please remove nearby-seismicity references or get that work merged first. Currently I can't start the application and get the following error:
```
ERROR in src/app/executive/executive.module.ts(17,46): error TS2307: Cannot find module './nearby-seismicity-pin/nearby-seismicity-pin.component'.
```

**Pull request merged**